.github/workflows/slint_tool_binary.yaml

```diff
@@ -25,7 +26,8 @@ on:
       features:
```

Comment:
I wonder why this is even an input of the workflow since this is not overriden anywhere.  Could be hardcoded.
(And if we were to override it we would need to keep in sync)

Reply:
I think eventually we should pass those values from the calling workflow, but atm that’s not working very well :(